docs/install.md

```diff
@@ -164,12 +164,14 @@ The standalone distribution will still download core and third-party plugins as
 
 You can launch workflows directly from [Seqera Platform](https://seqera.io/platform/) without installing Nextflow locally.
 
-Launching from Seqera Platform provides you with:
+Launching from Seqera provides you with:
 
 - User-friendly launch interfaces.
 - Automated cloud infrastructure creation.
 - Organizational user management.
 - Advanced analytics with resource optimization.
 
 Seqera Cloud Basic is free for small teams. Researchers at qualifying academic institutions can apply for free access to Seqera Cloud Pro.
-See the [Seqera Platform documentation](https://docs.seqera.io/platform) for tutorials to get started.
+See [Seqera Platform Cloud](https://docs.seqera.io/platform) to get started.
+
+If you have installed Nextflow locally, you can use the {ref}`nextflow auth <cli-auth>` command to authenticate with Seqera and automatically configure workflow monitoring.
```

Comment:
Intention is to also get `nextflow launch` in as part of this work (though still WIP and will be another PR). Would you rather update this once it's there?

Not sure if this would be better as an admonition? Though I know I tend to go a bit overboard with adding them everywhere.